RecoLocalTracker/SiPixelClusterizer/src/SiPixelFedCablingMapGPUWrapper.cc

```diff
@@ -22,11 +22,11 @@ SiPixelFedCablingMapGPUWrapper::SiPixelFedCablingMapGPUWrapper(SiPixelFedCabling
                                                                TrackerGeometry const& trackerGeom,
                                                                SiPixelQuality const *badPixelInfo):
   cablingMap_(&cablingMap),
-  fedMap(pixelgpudetails::MAX_SIZE), linkMap(pixelgpudetails::MAX_SIZE), rocMap(pixelgpudetails::MAX_SIZE),
-  RawId(pixelgpudetails::MAX_SIZE), rocInDet(pixelgpudetails::MAX_SIZE), moduleId(pixelgpudetails::MAX_SIZE),
-  badRocs(pixelgpudetails::MAX_SIZE), modToUnpDefault(pixelgpudetails::MAX_SIZE),
+  modToUnpDefault(pixelgpudetails::MAX_SIZE),
   hasQuality_(badPixelInfo != nullptr)
 {
+  cudaCheck(cudaMallocHost(&cablingMapHost, sizeof(SiPixelFedCablingMapGPU)));
+
   std::vector<unsigned int> const& fedIds = cablingMap.fedIds();
   std::unique_ptr<SiPixelFedCablingTree> const& cabling = cablingMap.cablingTree();
```

Comment:
not relevant for this PR, but wouldn't it be simpler to use a "dumb" pointer (`SiPixelFedCablingTree const *`) instead of a const referent to a `unique_ptr` ?

Reply:
I'd say it would be clearer to drop `const&` and take the `unique_ptr` by value as `cablingMap.cablingTree()` returns the `unique_ptr` by value
https://github.com/cms-patatrack/cmssw/blob/30e75bba952417d59e6a176c369b127f8cf6677d/CondFormats/SiPixelObjects/interface/SiPixelFedCablingMap.h#L35

Reply:
same for `fedIds` ?
`cablingMap.fedIds()` return an `std::vector` by value, so we could drop the `const&` there as well, and let the compiler move or even optimise it away